index.rst

```diff
@@ -118,68 +153,81 @@ to diagnose or characterize certain behaviour), and personal notebooks. The deta
 scope of this technote. Until the re-organisation is completed, tags will be made of the repo at least every ~6 months,
```

Comment:
Simon made a valid point that removing files from a branch in a repo does not necessarily remove the file from the repo. I think we should rephrase this a bit. Maybe something on the lines of:

Until the re-organisation is completed, tags will be made of the repo at least every 6 months or before/after major activities. After each release, user will be asked to review and possibly remove notebooks older than 1 year to make sure stale notebooks are not lingering alongside the main working branch.

Reply:
Replaced